index.ts

```diff
@@ -2,11 +2,13 @@ import ReactSelectorQuery from './src/resq'
 import { waitToLoadReact } from './src/waitToLoadReact'
 import { findReactInstance } from './src/utils'
 
-function doQuery(selector, method, element) {
+function doQuery(selector: any, method: any, element: any) {
+    // @ts-expect-error ts-migrate(2304) FIXME: Cannot find name 'global'.
```

Comment:
`global` is also a reference to `window`, so we can use that instead.

Reply:
Thanks!
Changed that to `window` and added type declarations for react stuff